lib/ical/property.js

```diff
@@ -211,7 +211,11 @@ ICAL.Property = (function() {
      * @return {Array|String}        Property value
      */
     getParameter: function(name) {
-      return this.jCal[PROP_INDEX][name];
+      if (this.jCal[PROP_INDEX][name]) {
```

Comment:
I believe in strict mode this will still give you a warning, you'd need to do `typeof this.jCal[PROP_INDEX][name] == undefined` or `name in this.jCal[PROP_INDEX]`. Maybe you can also make it a ternary if to keep it a one-liner.